manim/mobject/geometry.py

```diff
@@ -166,6 +166,10 @@ def position_tip(self, tip, at_start=False):
             handle = self.get_last_handle()
             anchor = self.get_end()
         tip.rotate(angle_of_vector(handle - anchor) - PI - tip.tip_angle)
+        tip.rotate(
+            PI / 2 - np.arccos(normalize(anchor)[2]),
```

Comment:
Just for the sake of testing whether it is a problem with the difference: I'd like to try something like

```suggestion
            0 if anchor[2] == 0 else PI / 2 - np.arccos(normalize(anchor)[2]),
```

Reply:
> Just for the sake of testing whether it is a problem with the difference: I'd like to try something like

Still the same result unfortunately